src/components/Navbar/SettingsDrawer/sample-verse-data.ts

```diff
@@ -0,0 +1,222 @@
+import Word from 'types/Word';
+
+// a hard coded sample verse
+// currently we use it in QuranFontSection, so we can show the preview of the verse
+// when use users are changing the font size, style, etc
+const verseSample = {
```

Comment:
this should live somewhere else. let's chat about it

Reply:
src/utils/sample-version.json

src/utils/verse.ts expose `getSampleVerse()`